From aea41ef70ec46726d81986a4eda4dedecfbdce38 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Tue, 7 Nov 2017 15:21:42 +0800
Subject: [PATCH 2739/5242] MLK-16727-4 ARM64: dts: fsl-imx8qxp-mek: change PD
 role as source only

commit  10ef7c96a7437b2d1105b56c5ac7e258cc82f8ed from
https://source.codeaurora.org/external/imx/linux-imx.git

At MEK hardware design, the PD is source only, so the software must
configure PTN5110 as source only, otherwise, there is an inrush higher
voltage at vbus when connects to PC due to EN_SRC is enabled at DRP
mode, and causes some PC are shut down.

BuildInfo:
- SCFW 8dcff26, IMX-MKIMAGE ea027c4b, ATF
- U-Boot 2017.03-imx_v2017.03_4.9.51_imx8_beta1+g6dc7b0f

Acked-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index fce42d6..9d732d3 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -650,8 +650,8 @@
 		max-snk-mv = <9000>;
 		max-snk-ma = <1000>;
 		op-snk-mw = <9000>;
-		port-type = "drp";
-		default-role = "sink";
+		port-type = "dfp";
+		default-role = "source";
 		status = "okay";
 	};
 };
-- 
1.7.9.5

